.circleci/config.yml

```diff
@@ -853,6 +863,72 @@ jobs:
           name: Run the check_upstream_chart script
           command: |
             ./script/chart_upstream_checker.sh << pipeline.parameters.CI_BOT_USERNAME >> << pipeline.parameters.CI_BOT_EMAIL >> << pipeline.parameters.CI_BOT_GPG >> << pipeline.parameters.CI_BOT_FORKED_CHARTS_DEPLOYKEY_FILENAME >> << pipeline.parameters.CHARTS_REPO_ORIGINAL >> << pipeline.parameters.BRANCH_CHARTS_REPO_ORIGINAL >> << pipeline.parameters.CHARTS_REPO_FORKED >> << pipeline.parameters.BRANCH_CHARTS_REPO_FORKED >> << pipeline.parameters.KUBEAPPS_REPO >> << pipeline.parameters.BRANCH_KUBEAPPS_REPO >> << pipeline.parameters.README_GENERATOR_REPO >>
+  report_srp:
+    environment:
+      <<: *common_envars
+    docker:
+      - image: cimg/go:<< pipeline.parameters.GOLANG_VERSION >>
+        auth:
+          username: $DOCKER_USERNAME
+          password: $DOCKER_PASSWORD
+    steps:
+      - checkout
+      - run:
+          name: Init submodules
+          command: |
+            git submodule update --init --recursive
+      - run:
+          name: Install and configure the SRP tool
+          command: |
+            wget https://vmwaresaas.jfrog.io/artifactory/srp-tools/srpcli/<< pipeline.parameters.SRP_VERSION >>/linux/srp
+            sudo chmod +x srp
+            sudo mv ./srp /usr/local/bin/
+            srp --version
+            srp config --srp-endpoint << pipeline.parameters.SRP_ENDPOINT >>
+            srp config auth --client-id $KUBEAPPS_SRP_CLIENT_ID --client-secret $KUBEAPPS_SRP_CLIENT_SECRET
+      # - run:
+      #     name: Install and configure the SRP observer tool
```

Comment:
It isn't working yet, but the team is working on providing a public endpoint for it. I'll send another PR once we have it.